### PR TITLE
Add audio mixer visualiser (Ctrl+F9)

### PR DESCRIPTION
**Pull request by smoogipoo**
Reimplements the visualiser which was removed in https://github.com/ppy/osu-framework/pull/4291

- [x] Depends on https://github.com/ppy/osu-framework/pull/4291

Very basic - I think it'd be useful to display the channel/mixer names in the future, but should be enough for now since we'll only have few channels/mixers.

### CODE REVIEW

File: osu.Framework/Input/FrameworkActionContainer.cs [hunk 1]
```diff
@@ -18,7 +18,8 @@ internal class FrameworkActionContainer : KeyBindingContainer<FrameworkAction>
             new KeyBinding(new[] { InputKey.Control, InputKey.F11 }, FrameworkAction.CycleFrameStatistics),
             new KeyBinding(new[] { InputKey.Control, InputKey.F10 }, FrameworkAction.ToggleLogOverlay),
             new KeyBinding(new[] { InputKey.Alt, InputKey.Enter }, FrameworkAction.ToggleFullscreen),
-            new KeyBinding(new[] { InputKey.F11 }, FrameworkAction.ToggleFullscreen)
+            new KeyBinding(new[] { InputKey.F11 }, FrameworkAction.ToggleFullscreen),
+            new KeyBinding(new[] { InputKey.Control, InputKey.F9 }, FrameworkAction.ToggleAudioMixerVisualiser),
```
Comment (peppy) [inline osu.Framework/Input/FrameworkActionContainer.cs:22]: Any reason for avoiding `Ctrl+F4`?

Comment (smoogipoo) [reply osu.Framework/Input/FrameworkActionContainer.cs:22]: No reason, this is what it initially was so I assumed it was suggested to be this value somewhere else. F4 sounds good.

Comment (bdach) [reply osu.Framework/Input/FrameworkActionContainer.cs:22]: Ctrl-F4 is `DocumentClose` on windows, fwiw.

Comment (peppy) [reply osu.Framework/Input/FrameworkActionContainer.cs:22]: Surprised we have that, even though it's probably never going to be used. I guess we should avoid F4 in that case. And maybe F5 as well - if we're supporting document close then we probably will end up supporting refresh via F5?

Comment (peppy) [reply osu.Framework/Input/FrameworkActionContainer.cs:22]: Let's just keep it as F9.

Comment (bdach) [reply osu.Framework/Input/FrameworkActionContainer.cs:22]: F4 will be used game-side for closing chat tabs. It's not that rare of a binding, on a quick check Firefox/Chrome/VS Code support it on windows. I think rider also does.

**Review by peppy — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Comment by Susko3** [comment]
This should be added to the wiki at https://github.com/ppy/osu-framework/wiki/Framework-Key-Bindings

**Comment by peppy** [comment]
I've added it

**Pull request merged**